Resources/doc/audit-log-entity-orm.md

```diff
@@ -1,6 +1,6 @@
 # Step 3:  Create audit_log entity mapping
 
-BaseAuditLog class does not provide ODM/ODM Mapping,
+BaseAuditLog class does not provide ORM/ODM Mapping,
```

Comment:
As you are fixing the typo, can you write it as 

`BaseAuditLog class does not provide any ORM/ODM Mapping,`

Reply:
Should I have written 'BaseAuditLog class does not provide any ORM or ODM Mapping' instead ?